examples/jsm/nodes/materials/nodes/StandardNode.js

```diff
@@ -32,7 +32,15 @@ StandardNode.prototype.build = function ( builder ) {
 
 	var code;
 
-	builder.define( this.clearCoat || this.clearCoatRoughness || this.clearCoatNormal ? 'PHYSICAL' : 'STANDARD' );
+	builder.define('PHYSICAL');
+
+	var useClearCoat = this.clearCoat;
```

Comment:
I loved your code, only here if you can preserve the previous logic, the code was writing thinking of using any of these inputs to active clearCoat.

```
this.clearCoat || this.clearCoatRoughness || this.clearCoatNormal
```